ansible/playbooks/restore-backup.yml

```diff
@@ -0,0 +1,106 @@
+---
+# This playbook can be run on staging to test the integrity of the backup, and
+# in production to set up a new production environment. Because it always needs
+# access to the production AWS bucket, when running on staging you need to add
+# "--vault-id production@prompt" to the run-playbook.sh call!
```

Comment:
This seems like it is easy to forget and not too important for everyone to remember. Can we create a small wrapper script which provides the right flags to Ansible for the user? Or provide an example invocation

Can you also document what `--vault-id` does? I have no idea :)

Also, can we document the prerequisites for this script? I assume a bootstrapped host where the playbook has been ran, but is that correct?

Reply:
> Also, can we document the prerequisites for this script? I assume a bootstrapped host where the playbook has been ran, but is that correct?
Yup, a functioning staging or production environment. Will add.

Reply:
> Since we use run-playbook.sh for all playbook runs, I'm not sure having to run another wrapper script (which is only used for this script) will be harder to forget than having to add a flag to the usual one

We could also hardcode the `restore-backup.yml` playbook name and automatically add the right flag?

Reply:
> We could also hardcode the restore-backup.yml playbook name and automatically add the right flag in run-playbook.sh?

Good point, that should be doable without "polluting" the script very much.